cumulus/pallets/collator-selection/src/mock.rs

```diff
@@ -95,7 +95,7 @@ impl pallet_balances::Config for Test {
 	type RuntimeHoldReason = RuntimeHoldReason;
 	type RuntimeFreezeReason = RuntimeFreezeReason;
 	type FreezeIdentifier = ();
-	type MaxHolds = ConstU32<0>;
+
```

Comment:
There seem to be a lot of these blank lines introduced. Should be able to remove them with an awk script

Reply:
hmm, nor local `cargo +nightly fmt` did not catch those: https://github.com/paritytech/polkadot-sdk/pull/2657/commits/248790a824671948af3506ac5c60a147d733ac12  neither "fmt bot" as I see

thank you, let me clean them

Reply:
@seadanda 
yeah, fmt ignores that, because it is ok,
but I fixed all touched files, thank you  again :)